docs/development/building/Dfu_Flash.md

```diff
@@ -0,0 +1,19 @@
+# Building DFU_Flash
+
+Pre-requisites: python3, python3-intelhex
+
+### Make and Flash via dfu-util:
+
+For Betaflight 4.5 and later use the command syntax:
+
+```
+make CONFIG=TARGETNAME dfu_flash
+```
+
+Where `TARGETNAME` is a specified valid flight-controller target.
+
+Example:
+
+```
+make CONFIG=MATEKF405AIO dfu_flash
```

Comment:
```suggestion
make MATEKF405AIO dfu_flash
```

Reply:
I tested the following: `make MATEKF405AIO dfu_flash` versus `make CONFIG=MATEKF405AIO dfu_flash`.

The former will try to flash `betaflight_4.5.2_STM32F405.dfu`, where as the latter will correctly try to flash `betaflight_4.5.2_STM32F405_MATEKF405AIO.dfu`.

Unless I am mistaken on the usage, i believe the `CONFIG=` is required -- please inform me.

Reply:
Yes I was wrong, these sub commands still use old style.

Reply:
so two options:
- merge documentation as-is, or
- someone fix make system.

 :crystal_ball: